### PR TITLE
Fix #2406:  Link to recommended system specs before purchase 

### LINKED ISSUE #2406: Link to recommended system specs before purchase
Move recommended system specs to it's own page, and link to it before the purchase flow. Near the download size info maybe

### PR DESCRIPTION
**Pull request by lewisgoddard**
Fixes #2406: Link to recommended system specs before purchase

This pull request is ready for review.

### CODE REVIEW

**Comment by lewisgoddard** [comment]
image

**Comment by cassidyjames** [comment]
Yeah I think I'd actually remove the size and 64-bit, and change that whole line into "System requirements" or something, and just link it to the installation guide section. I agree I don't want to break old links, plus there's other good info there about what they're gonna need to do to install.

**Comment by lewisgoddard** [comment]
image

File: index.php [hunk 1]
```diff
@@ -60,8 +60,8 @@
                         <div class="column">
                             <button type="submit" id="download" class="suggested-action"><?php echo ($already_paid) ? "Download elementary OS" : "Purchase elementary OS"; ?></button>
                             <p class="small-label">
-                                elementary OS <?php echo $config['release_version'] . ' ' . $config['release_title']; ?><br>
-                                <?php echo $config['release_size']; ?> | 64-bit
+                                elementary OS <?php echo $config['release_version'] . ' ' . $config['release_title']; ?> | <?php echo $config['release_size']; ?><br>
+                                <a href="docs/installation#recommended-system-specifications" target="_blank" rel="noopener">recommended system specifications</a>
```
Comment (cassidyjames) [inline index.php:64]:
```suggestion
                                <a href="docs/installation#recommended-system-specifications" target="_blank" rel="noopener">Recommended System Specifications</a>
```

**Pull request merged**